cfn.yaml

```diff
@@ -26,11 +26,11 @@ Parameters:
 Mappings:
   StageVariables:
     CODE:
-      UploadPath: reserved-paths/CODE/ios-live-app/
```

Comment:
Are the iOS apps not reading this path?

Reply:
Yes, that continues to work because we [allow](https://github.com/guardian/live-app-versions/pull/24/files#diff-914fd579658dc67679af7407e39e4894R81) the lambda to upload to `arn:aws:s3:::static-content-dist/${Path}*`. This is just a broadening of permissions which allows us to upload to any directory under `reserved-paths`. Thinking about it some more, perhaps `reserved-paths/CODE/*-live-app/` would have been better...